### PR TITLE
Dropped external subtitles

### PR DESCRIPTION
**Pull request by AndriiMoskvin**
We no longer going to support external subtitles - so I've dropped all references to them inside controls.

@aol-public/mobile-sdk-team: Ready for review.

[JIRA Issue](https://jira.ouroath.com/browse/OMSDK-810)

### CODE REVIEW

File: PlayerControls/sources/DefaultControlsViewController.swift [hunk 1]
```diff
@@ -208,8 +205,6 @@ public final class DefaultControlsViewController: ContentControlsViewController
                 airplayPipTrailingConstrains.isActive = !nextUIProps.pipButtonHidden && !nextUIProps.airplayButtonHidden
                 airplayEdgeTrailingConstrains.isActive = nextUIProps.pipButtonHidden && !nextUIProps.airplayButtonHidden
                 subtitlesAirplayTrailingConstrains.isActive = !nextUIProps.airplayButtonHidden
-                subtitlesEdgeTrailingConstrains.isActive = nextUIProps.airplayButtonHidden && nextUIProps.pipButtonHidden && !nextUIProps.settingsButtonHidden
-                subtitlesPipTrailingConstrains.isActive = nextUIProps.airplayButtonHidden && !nextUIProps.pipButtonHidden
```
Comment (RomanTysiachnik) [inline PlayerControls/sources/DefaultControlsViewController.swift]: Why delete 2/3 constraints for subtitles `Button`? =)

File: PlayerControls/sources/DefaultControlsViewController.swift [hunk 2]
```diff
@@ -55,7 +55,6 @@ public final class DefaultControlsViewController: ContentControlsViewController
     @IBOutlet private var liveIndicationView: UIView!
     @IBOutlet private var liveDotLabel: UILabel!
     
-    @IBOutlet private var visibleControlsSubtitlesConstraint: NSLayoutConstraint!
```
Comment (RomanTysiachnik) [inline PlayerControls/sources/DefaultControlsViewController.swift]:
``` swift
@IBOutlet private var ccTextLabel: UILabel!
```
Should be deleted too.

**Comment by RomanTysiachnik** [comment]
Rebase please.

File: fastlane/Fastfile [hunk 3]
```diff
@@ -1,7 +1,9 @@
-# All lines starting with a # are ignored when running `fastlane`
-
 fastlane_version '2.83.0'
 
+before_all do |lane, options|
+  xcversion(version: '9.3')
```
Comment (RomanTysiachnik) [inline fastlane/Fastfile]: What is this for?

Comment (AndriiMoskvin) [reply fastlane/Fastfile]: It is requirement for current controls to be built only with Xcode 9.3

**Pull request merged**